migration-engine/migration-engine-tests/tests/migrations/id.rs

```diff
@@ -107,8 +107,9 @@ fn length_prefixed_compound_primary_key(api: TestApi) {
     });
 }
 
+// Ignoring TiDB, unsupported drop primary key when the table's pkIsHandle is true.
 // TODO: ignore because not possible on cockroachdb. We would need a multi-step process there.
-#[test_connector(exclude(Vitess), exclude(CockroachDb))]
+#[test_connector(exclude(Vitess), exclude(CockroachDb, TiDB))]
```

Comment:
What does that mean exactly?
Is there an issue for this?

Reply:
I did some investigation and I learned that this problem is also caused by TIDB does not support deleting the primary key on the cluster index table.

However, for some historical reasons, if the primary key type is integer (`INT` or `BIGINT`) and the table is cluster index table. The `pkIsHandle` flag of the table will be set to `true`, which means that the table will use the **primary key** as the row [handle](https://en.wikipedia.org/wiki/Handle_(computing)).

Our R&D engineers are considering optimizing this error message.

Reply:
Is there an issue for this work? Might be good to link there so we can notice when this is fixed one day.